app/controllers/api/v1/cars.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CarsController < ApplicationController
+  before_action :set_cars, only: %i[show destroy]
+  before_action :set_user, only: %i[create destroy]
+
+  # GET /cars/1
+  def show
+    render json: @car
+  end
```

Comment:
Please add a check condition for when no car is found in the database. I.E, what happens when no data is returned?
```suggestion
  def show
    if car
      render json: car, status: :ok
    else
      render json: car.errors, status: :bad_request
    end
  end
```